src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.ts

```diff
@@ -450,22 +484,31 @@ export class AdditionalServiceModalComponent {
   }
 
   save() {
-    if (!this.isServiceAvailable && !this.form.controls['serviceName'].value) {
+    if (!this.serviceInfo.added && !this.form.controls['serviceName'].value) {
       this.alertService.error('Missing service name');
       return;
     }
+    
+    // If form value is not changed then return
+    if ((this.isServiceEnabled === this.form.controls['enabled'].value) && isEmpty(this.changedConfig) && isEmpty(this.advancedConfiguration)) {
+      this.toggleModal(false);
+      this.additionalServicesUtils.navToAdditionalServicePage(this.fromListPage, this.serviceInfo.process);
+      this.alertService.error('Nothing to save');  
```

Comment:
What is this error for?

Reply:
As I've already mentioned in the comment (above this code), this appears when user clicks on Save button without changing value of any field.